server/src/lib/artefacts.ts

```diff
@@ -0,0 +1,216 @@
+import { existsSync, mkdirSync } from 'fs';
+import { readFile, writeFile, unlink } from 'fs/promises';
+import * as path from 'path';
+import * as zlib from 'zlib';
+
+import { S3 } from '@aws-sdk/client-s3';
+import { config } from 'dotenv';
+
+config();
+
+const { PREFERRED_STORAGE_TYPE, AWS_S3_BUCKET_NAME } = process.env;
+const REGION = 'eu-central-1';
```

Comment:
This should probably be in the .env as well

Reply:
Yeah, fair enough